examples/react/start-workos/app/authkit/ssr/workos.ts

```diff
@@ -0,0 +1,39 @@
+import { WorkOS } from '@workos-inc/node';
+import { getConfig } from './config';
+import { lazy } from './utils';
+
+const VERSION = '0.0.1';
+
+/**
+ * Create a WorkOS instance with the provided API key and optional settings.
+ */
+export function createWorkOSInstance() {
+  // Get required API key from config
+  const apiKey = getConfig('apiKey');
+
+  // Get optional settings
+  const apiHostname = getConfig('apiHostname');
+  const apiHttps = getConfig('apiHttps');
+  const apiPort = getConfig('apiPort');
+
+  const options = {
+    apiHostname,
+    https: apiHttps,
+    port: apiPort,
+    appInfo: {
+      name: 'authkit-react-router',
```

Comment:
Same question about this `options.name`.

Reply:
Removing this as well.